runtime/compiler/optimizer/SequentialStoreSimplifier.cpp

```diff
@@ -1102,7 +1099,6 @@ static TR::TreeTop* generateArraycopyFromSequentialStores(TR::Compilation* comp,
       {
       switch(numBytes)
          {
-         case 2: opcode = TR::cstorei; break;
```

Comment:
I don't think this delete is right - we should handle the case of 2 bytes I think

Reply:
This was a mistake, should have changed `cstorei` to `sstorei` instead of not handling the whole 2 bytes case. Going to fix this in the new force push.